pypesto/petab/importer.py

```diff
@@ -739,7 +342,8 @@ def create_problem(
         A :class:`pypesto.problem.Problem` for the objective.
         """
         if objective is None:
-            objective = self.create_objective(**kwargs)
+            self.factory = self.create_objective_creator()
```

Comment:
Why are the names of the objects different?
Importer has them as `self.factory`, and they are in the `factory.py` file, but all of the classes are called `Creator`.
Wouldn't it be simpler if it's either only one or the other?

i.e. `self.factory = self.create_objective_factory()` and those are, for instance, `PetabSimulatorObjectiveFactory`

Reply:
Oh, I just saw the discussion you had with Daniel about the naming (about XFactory should only make X). 
Should in that case nothing be called factory, and all should be creators, including here?